widget/embedded/src/components/Quote/Quote.styles.ts

```diff
@@ -112,6 +112,13 @@ export const SummaryContainer = styled('div', {
     paddingTop: '$10',
```

Comment:
I think in this branch color for quote component (main part and expandable part) doesn't have enough contrast (different) at least for the dark theme. It's a comparision with previous version which seems better: 

![Recording 2023-11-15 at 11 34 14](https://github.com/rango-exchange/rango-client/assets/99180934/393b2188-fa96-417a-ad33-9ae0cbc49af4)


Also in both cases, it would be great if you could improve text colors which are not visible.

Reply:
fixed